guice-jpa/src/test/java/com/maxifier/guice/jpa/DBEntityManagerProviderTest.java

```diff
@@ -0,0 +1,67 @@
+package com.maxifier.guice.jpa;
+
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.metamodel.Metamodel;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Konstantin Lyamshin (2015-11-17 0:15)
+ */
+public class DBEntityManagerProviderTest extends org.testng.Assert {
```

Comment:
Why? Extend works perfect and takes aways problems with Idea auto import optimization.